database/index.js

```diff
@@ -0,0 +1,41 @@
+const Sequelize = require('sequelize');
+
+const sequelize = new Sequelize('yumnoms', 'root', '', {
+  host: 'localhost',
+  dialect: 'mysql',
+});
+
+sequelize
+  .authenticate()
+  .then(() => {
+    console.log('Connection established successfully');
+  });
+
+const Review = sequelize.define('review', {
+  review_id: { type: Sequelize.STRING },
```

Comment:
Everything looks great! You may consider removing all ID attribute from your schemas. These are generated automatically when you use sequelize.